deploy/helm/kubecf/templates/eirini.yaml

```diff
@@ -11,8 +11,17 @@ metadata:
   name: {{ .Release.Namespace }}:eirini-webhook
 rules:
 - apiGroups: ['admissionregistration.k8s.io']
+{{- if .Values.features.eirini.rbac.cluster_role.eirini_webhook.rules.resources }}
+  resources: {{ toJson .Values.features.eirini.rbac.cluster_role.eirini_webhook.rules.resources }}
+{{- else }}
   resources: ['mutatingwebhookconfigurations']
-  verbs:     ['*']
+{{- end }}
+
+{{- if .Values.features.eirini.rbac.cluster_role.eirini_webhook.rules.verbs }}
+  verbs: {{ toJson .Values.features.eirini.rbac.cluster_role.eirini_webhook.rules.verbs }}
+{{- else }}
+  verbs: ['*']
+{{- end }}
```

Comment:
As suggested on https://github.com/cloudfoundry-incubator/kubecf/issues/595, we don't want to mix the user-provided  ClusterRole rules with the static ones. The suggestion was to add an _extra_ ClusterRole, and bind it to the eirini ServiceAccount. This way it's clearer during runtime what the user provided and what KubeCF provided by default.

Reply:
Got it. Change the ClusterRole name to eirini_webhook_user_provided.